packages/web-components/fast-foundation/src/menu-item/menu-item.ts

```diff
@@ -16,6 +16,12 @@ import { MenuItemRole } from "./menu-item.options";
 
 export { MenuItemRole };
 
+/**
+ * Types of button appearance.
```

Comment:
This description doesn't seem accurate to me. Also, does this really need to be public?

Reply:
I'll update the comment, but men does import it